helper-classes/list-reader.h

```diff
@@ -17,14 +22,33 @@ class ListReader {
 
   protected:
     std::string _filename;
+    inline std::vector<std::string> GetStringListData() {
+      std::vector<std::string> data_list;
```

Comment:
Not four spaces? Highly recommend using [clang format](https://clang.llvm.org/docs/ClangFormat.html) so you no longer have to think about whitespace.

Reply:
This looks like a typo lol. Good catch.

Reply:
I've gone with your suggestion up above to integrate `clang-format` for source file formatting, which should resolve this issue.